src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const baseCost = 40 * days;
+
+  if (days >= 7) {
+    return baseCost - 50;
```

Comment:
Replace the magic number `50` with a named constant, such as `DISCOUNT_FOR_SEVEN_DAYS`, to make the code more understandable.